mod/calculated_metrics/lib/formula/wolfram/unknowns.rb

```diff
@@ -1,36 +1,24 @@
 module Formula
   class Wolfram < NestFormula
-    # Provide methods to handle "Unknown" input values for Wolfram formulas
+    # Provide methods to handle "Unknown" return values for Wolfram formulas
     module Unknowns
-      # what to do if an input value is unknown
-      # currently hard-coded.
-      # We probably want this sharkable
-      UNKNOWN_STRATEGY = :reject # reject:  calculated value is nil
-                                 # unknown: calculated value is unknown
-                                 # pass:    pass "Unknown" as String to the formula;
-                                 #          the formula has to deal with it
-
-      def handle_unknowns company, year
-        @unknown_input ||= Hash.new_nested Array
-        catch(:unknown) do
+      # Deals with the "unknown: result_unknown" option for Wolfram formulas.
+      # If the formula is supposed to return "Unknown" because of an
+      # "Unknown" input value, then we have skip the calculation for that answer and
+      # add the "Unknown" after we got the result from the Wolfram server
+      def handle_unknowns input_values, company, year
+        @unknown_result ||= Hash.new_nested Array
```

Comment:
You removed all this in your branch. I restored it. I didn't see a solution in your code for the problem describe in the comment above.